### PR TITLE
added models

### LINKED ISSUE #9: Models

### PR DESCRIPTION
**Pull request by jessicapalomo**
Closes #9.

### CODE REVIEW

File: server/db/models/requests.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  photo: {
+    type: Image,
+    required: false
+  },
+  resolution: {
+    type: String,
+    default: false
+  }
+});
+const Request = mongoose.model('Request', userSchema);
```
Comment (leopolicastro) [inline server/db/models/requests.js:21]: Is it a `userSchema` or `requestSchema`

File: server/db/models/requests.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  aidType: {
+    type: Array,
+    required: true
+  },
+  description: {
+    type: String,
+    required: false
+  },
+  photo: {
+    type: Image,
```
Comment (leopolicastro) [inline server/db/models/requests.js:13]: I don't believe there is an Image data type.  This should be saved as a `String`

File: server/db/models/users.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+const userSchema = new mongoose.Schema(
+  {
+    name: {
+      type: String,
+      required: true,
+      trim: true
+    },
+    dob: {
+      type: String,
+      required: true
```
Comment (leopolicastro) [inline server/db/models/users.js:12]: I would use the `Date` data type for this one. Makes it easier to work with dates.

**Review by leopolicastro — CHANGES_REQUESTED** [review]

File: server/db/models/requests.js (hunk 2 above)
Comment (jessicapalomo) [reply server/db/models/requests.js:13]: Thanks, was wondering

File: package.json [hunk 4]
```diff
@@ -7,8 +7,11 @@
     "concurrently": "^5.3.0",
     "express": "^4.17.1",
     "mongodb": "^3.6.2",
-    "mongoose": "^5.10.9",
-    "morgan": "^1.10.0"
+    "mongoose": "^5.11.8",
+    "morgan": "^1.10.0",
+    "react": "^17.0.1",
+    "set-cookie-parser": "^2.4.6",
```
Comment (leopolicastro) [inline package.json:13]: Which package is set cookie parser?

**Review by leopolicastro — APPROVED** [review]

**Review by leopolicastro — APPROVED** [review]

**Pull request merged**